### DISCUSSION TITLE
client fetch error

### DISCUSSION THREAD

**Discussion by PatgioK** [nextauthjs/next-auth#5445] [Help]
### Question 💬

Hello, I'm new to using next and nextauth.
I also created a stack overflow question here: https://stackoverflow.com/questions/73875657/next-auth-client-fetch-error-already-configured-env
I've since downgraded next to 12.2.5 (which seems to fixed the dependency errors) to follow a tutorial I was watching but still have the same error.

When I try to login to my app at `http://localhost:3000/login` I get a blank page that just says Error localhost:3000. 

The error in the console is
`[next-auth][error][CLIENT_FETCH_ERROR] 
https://next-auth.js.org/errors#client_fetch_error NetworkError when attempting to fetch resource. 
Object { error: {…}, url: "/api/auth/providers", message: "NetworkError when attempting to fetch resource." }
`
I've double and triple checked I've configured the .env. I tried 
`NEXTAUTH_URL=http://localhost:3000 `
and 
`NEXTAUTH_URL='http://localhost:3000'`

my /pages/api/auth/[...nextauth].ts:
```
import { NextApiHandler } from "next";
import NextAuth from "next-auth";
import EmailProvider from "next-auth/providers/email";
import { PrismaAdapter } from "@next-auth/prisma-adapter"
import { PrismaClient } from "@prisma/client";

const prisma = new PrismaClient();
const authHandler: NextApiHandler = (req, res) => NextAuth(req, res, options);
export default authHandler;

const options = {
  providers: [
    EmailProvider({
      server: {
        host: process.env.SMTP_HOST,
        port: Number(process.env.SMTP_PORT),
        auth: {
          user: process.env.SMTP_USER,
          pass: process.env.SMTP_PASSWORD,
        },
      },
      from: process.env.SMTP_FROM,
    }),
  ],
  adapter: PrismaAdapter(prisma),
  secret: process.env.SECRET,
};
```

my repo is https://github.com/PatgioK/ja-clone

Any help is appreciated! Thanks

### How to reproduce ☕️

clone repo
add .env file to root and add `NEXTAUTH_URL=http://localhost:3000 `
npm install
npm run dev
go to http://localhost:3000/login
click login button

### Contributing 🙌🏽

No, I am afraid I cannot help regarding this

**Comment by balazsorban44** [comment]
I could not reproduce it locally. `NetworkError` hints at a different error that comes from your browser. Maybe a proxy, browswer extension? Could you check another browser/incognito mode/device/network to confirm? This is unlikely a bug in NextAuth.js.

**Reply by PatgioK** [reply]
Thanks for the reply,
I tried it on chrome and it works fine. It does not seem to work on Firefox or incognito in Firefox. 

I also tried downgrading to nextauth to 4.0.5 and still doesn't seem to work on Firefox.

**Reply by cynthia-n** [reply]
> Thanks for the reply, I tried it on chrome and it works fine. It does not seem to work on Firefox or incognito in Firefox.
> 
> I also tried downgrading to nextauth to 4.0.5 and still doesn't seem to work on Firefox.

Have you solved it
